search/views.py

```diff
@@ -64,7 +67,18 @@ def do_search(request, course_id=None):
                 raise InvalidPageSize(_('Invalid page size of {page_size}').format(page_size=size))
 
             if "page_index" in request.POST:
-                from_ = int(request.POST["page_index"]) * size
+                page = int(request.POST["page_index"])
+                from_ = page * size
+
+        #Google Analytics - log search request
```

Comment:
Note this call is emtting events just to the tracking logs. You have to make a separate call to emit events to Segment.io (which, in turn, passes them on to google analytics).